src/main/java/by/exadel/internship/dto/formDTO/FormRegisterDTO.java

```diff
@@ -6,13 +6,14 @@
 import lombok.NoArgsConstructor;
 import lombok.experimental.SuperBuilder;
 
+
 @Data
 @SuperBuilder
 @NoArgsConstructor
 @AllArgsConstructor
 public class FormRegisterDTO {
 
-    private String name;
+    private String firstName;
```

Comment:
Add please validation annotations for required fields.

Reply:
+ regex for email

Reply:
@kzaretskaya @ShirshovAlex  The validation Library has special annotation @Email, so there is no need to use regular expressions.

Reply:
we'll do it later

Reply:
Create, please, in Trello appropriate ticket for keeping in mind this.